pallets/manta-pay/Cargo.toml

```diff
@@ -62,30 +58,33 @@ rand = { version = "0.8.4", default-features = false, optional = true }
 tempfile = { version = "3.3.0", optional = true }
 
 # substrate dependencies
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false, optional = true }
-frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
-frame-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.16", default-features = false, optional = true }
```

Comment:
is making the `.git` explicit necessary? I'm thinking it's obviously a git link due to the `git = ` already

Reply:
This is tricky. If I run `cargo b --release`, the `Cargo.lock` won't be updated. But while I run `cargo test` under `manta-pay`, it will be updated due to lacking of the tag `.git`.

Reply:
Don't really understand the different behavior since the short url should resolve to the .git one anyway, but there's no harm being explicit